src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -357,8 +369,20 @@ def create_nfs_cluster(self, export_type, size):
                 return r, out, err
 
         self.create_empty_rados_obj()
-        #TODO Check if cluster exists
-        #TODO Call Orchestrator to deploy cluster
+
+        if self.check_cluster_exists():
+            return -errno.EINVAL,"", "Cluster already exists"
```

Comment:
hm. the mon_thrasher will duplicate random mon commands. If you want to survive the mon_thrasher, you should make your commands idempotent.

Reply:
Done